db/warehouse/migrate/20231122185546_add_hmis_auto_exit_config.rb

```diff
@@ -0,0 +1,13 @@
+class AddHmisAutoExitConfig < ActiveRecord::Migration[6.1]
+  def change
+    add_column :Exit, :auto_exited, :datetime, null: true
+
+    create_table(:hmis_auto_exit_configs) do |t|
+      t.integer :length_of_absence_days, null: false, default: 30
+      t.integer :project_type
+      t.references :organization, foreign_key: { to_table: 'Organization' }
+      t.integer :project_id, foreign_key: { to_table: 'Project' }
```

Comment:
Did you mean to make this one reference too?

Reply:
Whoops, yes I did. Fixed!